internal/riverinternaltest/riverinternaltest.go

```diff
@@ -0,0 +1,117 @@
+package riverinternaltest
```

Comment:
This and all files in the diff below this one were copied basically wholesale from the main River project.